templates/legal/confidentiality-agreement.md

```diff
@@ -0,0 +1,620 @@
+---
+wrapper_template: "legal/_base_legal_markdown.html"
```

Comment:
This page throws an error:
https://canonical-com-1792.demos.haus/legal/confidentiality-agreement